src/features/projects/components/maps/Home.tsx

```diff
@@ -42,15 +42,37 @@ export default function Home({
     setViewPort(newViewport);
   }, []);
 
-  const markerProps = {
-    searchedProject,
-    setPopupData,
-    popupData,
-    isMobile,
-  };
+  const topApprovedProjects = searchedProject.filter(
+    (project) =>
+      project.properties.isTopProject && project.properties.isApproved
+  );
+
+  const topUnapprovedProjects = searchedProject.filter(
+    (project) =>
+      project.properties.allowDonations &&
+      !(project.properties.isTopProject && project.properties.isApproved)
+  );
+
+  const notDonatableProjects = searchedProject.filter(
+    (project) =>
+      !project.allowDonations &&
```

Comment:
Why do you use `project.allowDonations` here while using `project.properties.allowDonations` above?

Reply:
Oh I forgot to make the correction here